docker/archlinux.dockerfile

```diff
@@ -0,0 +1,75 @@
+# syntax=docker/dockerfile:1.4
+# artifacts: true
+# platforms: linux/amd64
+ARG BASE=archlinux
+ARG TAG=base-devel
+FROM ${BASE}:${TAG} AS sunshine-base
+
+FROM sunshine-base as sunshine-build
+
+ARG BUILD_VERSION
+ARG COMMIT
+ARG GITHUB_SHA=$COMMIT
+# note: BUILD_VERSION may be blank, COMMIT is also available
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Need to determine why workflow doesn't fail when a command fails

Possibly due to using heredocs? Or the shebang within the heredocs?

Reply:
Seems to be an issue in heredocs... Will use `set -e` to resolve for now.